mdn/mdn-fiori
# Migrate styleguide pages to Storybook

Changes:
- Switch from `@storybook/html` to `@storybook/react`
- Delete old example files 
- Move content from https://mdn.github.io/mdn-fiori/ to Storybook docs

To test: 
- Run `yarn` and then `yarn start`
- Go to http://localhost:6006/ and should see something like the following: 
<img width="1320" alt="Screen Shot 2020-05-17 at 11 28 42 AM" src="https://user-images.githubusercontent.com/650/82152791-b07ca680-9831-11ea-82dd-225e5ccdf8dc.png">
